helpers/config.py

```diff
@@ -333,59 +392,42 @@ def get_config_template(cls):
                 "listen_addresses = '*'",
                 'max_connections = 100',
             ]),
-            'custom_secret_keys': Config.FALSE,
-            'enketo_api_token': binascii.hexlify(os.urandom(60)).decode(
-                'utf-8'),
-            'enketo_encryption_key': binascii.hexlify(os.urandom(60)).decode(
-                'utf-8'),
-            'django_secret_key': binascii.hexlify(os.urandom(50)).decode(
-                'utf-8'),
-            # default value from enketo. Because it was not customizable before
-            # we want to keep the same value when users upgrade.
-            'enketo_less_secure_encryption_key': 'this $3cr3t key is crackable',
-            'use_backup': Config.FALSE,
-            'use_wal_e': Config.FALSE,
-            'kobocat_media_schedule': '0 0 * * 0',
-            'mongo_backup_schedule': '0 1 * * 0',
-            'postgres_backup_schedule': '0 2 * * 0',
-            'redis_backup_schedule': '0 3 * * 0',
-            'aws_backup_bucket_name': '',
-            'aws_backup_yearly_retention': '2',
-            'aws_backup_monthly_retention': '12',
-            'aws_backup_weekly_retention': '4',
-            'aws_backup_daily_retention': '30',
-            'aws_mongo_backup_minimum_size': '50',
-            'aws_postgres_backup_minimum_size': '50',
-            'aws_redis_backup_minimum_size': '5',
-            'aws_backup_upload_chunk_size': '15',
-            'aws_backup_bucket_deletion_rule_enabled': Config.FALSE,
-            'backend_server_role': 'primary',
-            'use_letsencrypt': Config.TRUE,
-            'proxy': Config.TRUE,
-            'https': Config.TRUE,
-            'nginx_proxy_port': Config.DEFAULT_PROXY_PORT,
-            'exposed_nginx_docker_port': Config.DEFAULT_NGINX_PORT,
-            'expose_backend_ports': Config.FALSE,
+            'postgres_user': 'kobo',
             'postgresql_port': '5432',
-            'mongo_port': '27017',
-            'redis_main_port': '6379',
+            'primary_backend_ip': primary_ip,
+            'private_domain_name': 'kobo.private',
+            'proxy': True,
+            'public_domain_name': 'kobo.local',
+            'raven_settings': False,
+            'redis_backup_schedule': '0 3 * * 0',
             'redis_cache_port': '6380',
-            'local_installation': Config.FALSE,
-            'block_common_http_ports': Config.TRUE,
-            'npm_container': Config.TRUE,
-            'mongo_root_username': 'root',
-            'mongo_root_password': Config.generate_password(),
-            'mongo_user_username': 'kobo',
-            'mongo_user_password': Config.generate_password(),
+            'redis_main_port': '6379',
             'redis_password': Config.generate_password(),
-            'uwsgi_workers_start': '1',
-            'uwsgi_workers_max': '2',
+            'review_host': True,
+            'server_role': '',
```

Comment:
it really does not matter. Because it will be initialized by users' choice whatever the default.
You are right, let's keep it at `frontend` by default.